[2025-03-25T20:00:04.034+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T20:00:04.057+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2025-03-25T19:30:00+00:00 [queued]>
[2025-03-25T20:00:04.064+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2025-03-25T19:30:00+00:00 [queued]>
[2025-03-25T20:00:04.064+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2025-03-25T20:00:04.073+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_all_data> on 2025-03-25 19:30:00+00:00
[2025-03-25T20:00:04.085+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=99) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T20:00:04.087+0000] {standard_task_runner.py:72} INFO - Started process 112 to run task
[2025-03-25T20:00:04.088+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelamento', 'get_all_data', 'scheduled__2025-03-25T19:30:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmpmu97jtsg']
[2025-03-25T20:00:04.093+0000] {standard_task_runner.py:105} INFO - Job 64: Subtask get_all_data
[2025-03-25T20:00:04.178+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelamento.get_all_data scheduled__2025-03-25T19:30:00+00:00 [running]> on host 6049b9215306
[2025-03-25T20:00:04.309+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelamento' AIRFLOW_CTX_TASK_ID='get_all_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-25T19:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-25T19:30:00+00:00'
[2025-03-25T20:00:04.311+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-25T20:00:04.322+0000] {inaturalist_dag.py:150} INFO - Getting data from keyword: atropelamento
[2025-03-25T20:00:04.348+0000] {inaturalist_dag.py:41} INFO - Maximum id 264497584 for the request
[2025-03-25T20:00:07.449+0000] {inaturalist_dag.py:58} INFO - Got 0 results
[2025-03-25T20:00:07.454+0000] {inaturalist_dag.py:158} INFO - No more observations found, stopping.
[2025-03-25T20:00:07.477+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-25T20:00:07.483+0000] {python.py:240} INFO - Done. Returned value was: []
[2025-03-25T20:00:07.517+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-25T20:00:07.519+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_inaturalist_data_atropelamento, task_id=get_all_data, run_id=scheduled__2025-03-25T19:30:00+00:00, execution_date=20250325T193000, start_date=20250325T200004, end_date=20250325T200007
[2025-03-25T20:00:07.575+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T20:00:07.601+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
